content/articles/leave-a-review.markdown

```diff
@@ -0,0 +1,22 @@
+---
+title: Leave a Review
+excerpt: At DNSimple, we love hearing what our users have to say!
+categories:
+- DNSimple
+---
+
+Do you have something you want to share about DNSimple? Do you want to shout our praises from a rooftop? Would you like to make sure others know what we're all about?
+
+Here at DNSimple, we love hearing what our users have to say! 
```

Comment:
What do you think about using "customers" instead of "users"?

```suggestion
Here at DNSimple, we love hearing what our customers have to say! 
```

Reply:
Sounds good to me!